apps/widget/src/app/ModelAssessment.tsx

```diff
@@ -42,49 +40,20 @@ export class ModelAssessment extends React.Component {
       };
     }
 
-    const dataset: IDataset = {
-      categoricalMap: modelData.categoricalMap,
-      classNames: modelData.classNames,
-      featureNames: modelData.featureNames,
-      features: modelData.features,
-      sensitiveFeatures: modelData.sensitiveFeatures,
-      trueY: modelData.trueY
-    };
-
-    const modelExplanationData: IModelExplanationData = {
-      modelClass: "modelClass" in modelData ? modelData.modelClass : "blackbox",
-      precomputedExplanations: {
-        ebmGlobalExplanation: modelData.ebmData,
-        globalFeatureImportance: modelData.globalExplanation,
-        localFeatureImportance: modelData.localExplanations
-      },
-      predictedY: modelData.predictedY,
-      probabilityY: modelData.probabilityY
-    };
-
-    const causalAnalysisData: ICausalAnalysisData = {
-      global: modelData.globalCasualAnalysis,
-      local: modelData.localCasualAnalysis
-    };
-
     return (
       <ModelAssessmentDashboard
-        causalAnalysisData={causalAnalysisData}
-        dataset={dataset}
-        modelExplanationData={modelExplanationData}
+        {...modelData}
         requestPredictions={requestPredictionsMethod}
         requestDebugML={requestDebugMLMethod}
         requestMatrix={requestMatrixMethod}
         requestMetrics={requestMetricsMethod}
         requestImportances={requestImportancesMethod}
-        localUrl={modelData.localUrl}
-        locale={modelData.locale}
-        theme={modelData.theme}
-        supportedBinaryClassificationPerformanceKeys={
-          modelData.classification_methods
-        }
-        supportedRegressionPerformanceKeys={modelData.regression_methods}
-        supportedProbabilityPerformanceKeys={modelData.probability_methods}
+        localUrl={config.baseUrl}
+        locale={config.locale}
+        theme={undefined}
+        supportedBinaryClassificationPerformanceKeys={[]}
```

Comment:
this only exists for fairness, so perhaps you want to remove it...

Reply:
removed